build/build-docker-deploy.sh

```diff
@@ -11,7 +11,7 @@ set -euo pipefail
 # itself inside of docker passing "docker" as the argument causing the
 # commands in the if-branch to be executed within the docker
 # container.
-if [ "${1:-}" = "docker" ]; then
+if [ "${1-}" = "docker" ]; then
```

Comment:
The former defines the LHS to the RHS, while the latter only evaluates to
the RHS is the LHS is undefined.
On Jul 29, 2015 6:57 PM, "Peter Mattis" <notifications@github.com> wrote:

> In build/build-docker-deploy.sh
> <https://github.com/cockroachdb/cockroach/pull/1841#discussion_r35821674>:
>
> > @@ -11,7 +11,7 @@ set -euo pipefail
> >  # itself inside of docker passing "docker" as the argument causing the
> >  # commands in the if-branch to be executed within the docker
> >  # container.
> > -if [ "${1:-}" = "docker" ]; then
> > +if [ "${1-}" = "docker" ]; then
>
> What's the difference between ${1:-} and ${1-}?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cockroachdb/cockroach/pull/1841/files#r35821674>.
>